test/unit_tests/arkode/CXX_serial/ark_test_getjac.cpp

```diff
@@ -168,6 +170,26 @@ int main(int argc, char* argv[])
   // SUNDIALS context object for this simulation
   sundials::Context sunctx;
 
+  // Comparison tolerance
```

Comment:
Unlike CVODE(S) and IDA(S), this does not adjust the integrator tolerances based on the SUNDIALS realtype precision.  Should it?

Reply:
ARKODE has a bit more accuracy for two of entries in the Jacobian than CVODE(S) and IDA(S) and this is enough to pass the test without modifying the tolerances but all the tests should use the same values.